cob_cartesian_controller/include/cob_cartesian_controller/cartesian_controller.h

```diff
@@ -39,6 +39,18 @@
 #include <tf/transform_broadcaster.h>
 #include <tf/transform_listener.h>
 
+#include <cob_cartesian_controller/CartesianControllerAction.h>
+#include <actionlib/server/simple_action_server.h>
+#include <string.h>
+
+typedef actionlib::SimpleActionServer<cob_cartesian_controller::CartesianControllerAction> tSAS_CartesianControllerAction;
+
+struct action_type{
+        std::string name;
+        bool rotateOnly;
+        std::string profile;
+        double x, y, z, roll, pitch, yaw, vel, accl;
+};
```

Comment:
- this should go into a separate `x_data_types.h` file
 - `ActionType` (not `action_type`)
 - `rotate_only` (not `rotateOnly`)
 - can you further group x, y, z, roll, pitch, yaw into a `geometry_msgs::Pose` or `geometry_msgs::Transform`?